remote_config/CMakeLists.txt

```diff
@@ -36,7 +36,7 @@ set(flatbuffer_schemas
 set(FLATC_ARGS "${FLATBUFFERS_FLATC_SCHEMA_EXTRA_ARGS}")
 build_flatbuffers("${flatbuffer_schemas}"
                   ""
-                  "remote_config_generate_request_fbs"
```

Comment:
This seems to be reverting a github-first fix.

Reply:
Yeah, looks like it.